### ISSUE TITLE
VoxelCorgi.gltf is not displayed

### ISSUE THREAD

**Issue by cx20** [Reon90/redcube#9]
I tried to display [VoxelCorgi.gltf](https://github.com/cx20/jsdo-static-contents/blob/master/models/gltf/2.0/VoxelCorgi/glTF_merge/VoxelCorgi.gltf) with RedCube.js v2.0.5.
However, it was not displayed.
[RedCube.js + Voxelcorgi.gltf result:](http://jsdo.it/cx20/qnqM)
```
[.WebGL-000001C702EE5CF0]GL ERROR :GL_INVALID_OPERATION : glDrawArrays: attempt to access out of range vertices in attribute 6
```

The following is the display result in other libraries.
[Three.js + Voxelcorgi.gltf result:](http://jsdo.it/cx20/ahsW)
![image](https://user-images.githubusercontent.com/1833979/50590461-7b86e280-0ece-11e9-8856-fbbba8401c4d.png)

**Comment by Reon90** [comment]
@cx20 was fixed in 2.0.6

**Comment by cx20** [comment]
I confirmed that corgi will be displayed in the latest version.
[RedCube.js + Voxelcorgi.gltf result:](http://jsdo.it/cx20/gLgE)
![image](https://user-images.githubusercontent.com/1833979/50773999-ae1d4a80-12d5-11e9-8a8c-cc9238b47ae7.png)

**Comment by cx20** [comment]
@Reon90 When I changed the version of RedCube.js from 2.0.5 to 2.0.6 it seems that another model is no longer displayed correctly.

[RedCube.js 2.0.5 + trex_running.gltf result:](http://jsdo.it/cx20/Ev8K)
![image](https://user-images.githubusercontent.com/1833979/50774884-02c1c500-12d8-11e9-8868-8b7eabdc19a7.png)

[RedCube.js 2.0.6 + trex_running.gltf result:](http://jsdo.it/cx20/cnzb)
![image](https://user-images.githubusercontent.com/1833979/50774906-15d49500-12d8-11e9-9b7c-9b9eb361af5a.png)

**Comment by Reon90** [comment]
@cx20 was fixed in 2.0.7

**Comment by cx20** [comment]
I confirmed that the problem of T-Rex was resolved in 2.0.7.
[RedCube.js 2.0.7 + trex_running.gltf result:](http://jsdo.it/cx20/sUGA)
![image](https://user-images.githubusercontent.com/1833979/50907113-eb1a4600-1469-11e9-9b16-74d8fa787947.png)
